mobilecoind-json/src/data_types.rs

```diff
@@ -261,6 +340,523 @@ impl From<&mc_mobilecoind_api::SendPaymentResponse> for JsonTransferResponse {
     }
 }
 
+#[derive(Deserialize, Serialize)]
+pub struct JsonOutlay {
+    pub value: u64,
```

Comment:
Is this consistent with the earlier issue of javascript int capacity? Should we instead store all u64 fields as string?

Reply:
@rjwalters is correct, amounts should be Strings due to JS ints being less than 64 bit.

Reply:
Thanks for pointing this out - fixed up throughout!